gef.py

```diff
@@ -2921,7 +2921,7 @@ def stop(self):
         msg = []
         if is_x86_32():
```

Comment:
Could we replace this with something like `current_arch.uses_stack_args()` or better yet `current_arch.get_call_arg`?